src/main/java/pro/sky/course_work2kurs/controller/JavaQuestionController.java

```diff
@@ -0,0 +1,36 @@
+package pro.sky.course_work2kurs.controller;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import pro.sky.course_work2kurs.model.Question;
+import pro.sky.course_work2kurs.service.JavaQuestionService;
+
+import java.util.Collection;
+
+@RestController
+@RequestMapping("/exam/java")
+public class JavaQuestionController {
+
+    private final JavaQuestionService javaQuestionService;
```

Comment:
```suggestion
    private final QuestionService javaQuestionService;
```
по интерфейсу инъекцию надо делать, говорил про это на разборе